apps/website/src/pages/docs/dropdownmenu.mdx

```diff
@@ -14,12 +14,70 @@ import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-dropdownmenu' />
-
 <LiveExample src='DropdownMenu.main.tsx'>
   <AllExamples.DropdownMenuMainExample client:load />
 </LiveExample>
 
+A dropdown menu is a control element meant to allow the user to select an option from a list. This is a list of options contained within a collapsible container. These are mostly found in forms and dialogs. When it is clicked, it expands into a full list of items. This list can be scrollable if there are a lot of options, to avoid covering up too much of the interface. It collapses back into a single line when the user either selects an option in the list, or clicks outside of it.
```

Comment:
{...} container and are mostly found {...}

Reply:
Thanks. I have updated this.